addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/MainTVServerService.java

```diff
@@ -130,7 +132,7 @@ public void run() {
 
                     updateResourceState("MainTVAgent2", "GetCurrentBrowserURL", null);
                 } catch (Exception e) {
-                    logger.debug("Exception during poll : {}", e);
+                    reportError("Error occurred during poll", e);
```

Comment:
Thanks for testing!

@paulianttila now I fully understand what you wrote, you fear that setting the tv offline in case of exceptions might break later if the exception handling in `UpnpIOServiceImpl` is changed. 

I don't think that that is very likely to happen soon, because that will be a breaking change for a lot of locations because a decent value is now returned and an exception logged; to change that to an exception being thrown / forwarded feels like a serious breaking change to me.

In my opinion the proper handling of exception given the current situation is more important than anticipating on a potential future breaking change. 

Can you agree with me on that?